backend/src/main/java/elbin_bank/issue_tracker/issue/domain/Issue.java

```diff
@@ -0,0 +1,43 @@
+package elbin_bank.issue_tracker.issue.domain;
+
+import elbin_bank.issue_tracker.common.domain.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table("issue")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class Issue extends BaseEntity {
+
+    @Id
+    private Long id;
+
+    @Column("writer_id")
+    private Long authorId;
+
+    @Column("milestone_id")
+    private Long milestoneId;
+
+    private String title;
+
+    @Column("contents")
+    private String contents;
+
+    @Column("is_closed")
+    private Boolean isClosed;
+
+    @Column("file_path")
+    private String filePath;
+
+    public boolean isClosed() {
+        return Boolean.TRUE.equals(isClosed);
+    }
```

Comment:
위의 isClosed에 primitive type을 사용할 수 있다면 다음과 같이 처리 가능하겠죠?
```suggestion
    public boolean isClosed() {
    return closed;
}
```